packages/mask/src/components/shared/ApplicationBoard.tsx

```diff
@@ -319,49 +336,58 @@ export function ApplicationBoard({ secondEntries, secondEntryChainTabs }: MaskAp
     ]
 
     return (
-        <>
-            {secondEntryChainTabs?.length ? (
-                <div className={classes.abstractTabWrapper}>
-                    <NetworkTab
-                        chainId={chainId}
-                        setChainId={setChainId}
-                        classes={classes}
-                        chains={secondEntryChainTabs}
+        <TargetChainIdContext.Provider>
+            <AllProviderTradeContext.Provider>
+                {secondEntryChainTabs?.length ? (
+                    <div className={classes.abstractTabWrapper}>
+                        <NetworkTab
+                            chainId={chainId}
+                            setChainId={setChainId}
+                            classes={classes}
+                            chains={secondEntryChainTabs}
+                        />
+                    </div>
+                ) : null}
+                <section className={classes.applicationWrapper}>
+                    {(secondEntries ?? firstLevelEntries).map(
+                        ({ title, img, onClick, supportedChains, hidden, walletRequired }, i) =>
+                            (!supportedChains || supportedChains?.includes(chainId)) && !hidden ? (
+                                <div
+                                    className={classNames(
+                                        classes.applicationBox,
+                                        walletRequired && !selectedWallet ? classes.disabled : '',
+                                    )}
+                                    onClick={onClick}
+                                    key={i}>
+                                    <img src={img} className={classes.applicationImg} />
+                                    <Typography className={classes.title} color="textPrimary">
+                                        {title}
+                                    </Typography>
+                                </div>
+                            ) : null,
+                    )}
+                </section>
+                {isClaimAllDialogOpen ? (
+                    <ClaimAllDialog open={isClaimAllDialogOpen} onClose={onClaimAllDialogClose} />
+                ) : null}
+                {isSavingsDialogOpen ? (
+                    <SavingsDialog
+                        open={isSavingsDialogOpen}
```

Comment:
```suggestion
                        open
```

Please clean up `open` prop on other `Dialog` in this file as well, that will make code cleaner.